### ISSUE TITLE
[V3] Implement new Concept Exercise: Template Strings and Ternary

### ISSUE THREAD

**Issue by junedev** [exercism/javascript#1009]
## Getting started
**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md)
- [What are concepts and how they are structured?](https://github.com/exercism/docs/blob/main/anatomy/tracks/concepts.md)
- [What are concept exercises and how they are structured?](https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md)

Please also watch the following video:
- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

See the documentation _above_ (general documentation), as well as [How to implement a Concept Exercise in JavaScript](https://github.com/exercism/javascript/blob/main/reference/implementing-a-concept-exercise.md).

## Goal
The goal of this exercise is to introduce template strings (also called template literals or string interpolation) and the ternary operator.

## Concepts
The following concept pages need to be created. You can combine the two introduction.md files from the concepts and use that as introduction.md for the concept exercise. No need to create different content at this point.
* `template-strings`
* `conditionals-ternary`

The ternary operator is included here because it probably does not make sense to have a separate concept exercise for it and it might be useful in the context of template strings. Additionally the `conditionals` concept/exercise was already quite "full".

## Learning Objectives
In the concepts the student should learn about the following topics and then practice them in the concept exercise.

* How to write a simple template string (point out that backticks are different from single quotes)
* How to use template strings to create multi-line strings
* How to do interpolation in template strings (explain that non-string values are converted to strings and that expressions are evaluated)
* How to use toFixed to format numbers
* How to use the conditional operator/[ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

## Out of Scope
* Nested and tagged template strings
* [Conditional Chains](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator#conditional_chains) (could also be part of "about" but since they are not very readable I would rather not mention them explicitly, people can still read about it on the general MDN page)

## About.md of the Concept
Here some ideas of what could be mentioned.

* Unlike C or Go (printf), JS does not have some built-in classical formatting library for processing something like e.g. `%9.2f` to format a number 
* Pitfall: rounding applied by toFixed is not mathematically correct in all cases

## Prerequisites
* `strings`
* `conditionals`
* `type-conversion`
* `type-coercion`

At this point, the student already knows about truthy and falsy. See [this list](https://github.com/exercism/javascript/issues/951#issuecomment-778834953) for details on the learning curve we are aiming for.

## Exercise Idea
[Swift Custom Signs Exercise](https://github.com/exercism/swift/blob/main/exercises/concept/custom-signs/.docs/instructions.md) could serve as inspiration for the story but it needs to be changed a bit.

* Omit the part with characters.
* Adapt the last task to include some calculation in the template string.
* Add a task that utilizes the ternary operator.

## Help
You can choose to do this solo-style, or collaborate with multiple people on this. The suggested approach is to 
1. **First accept this issue by saying "I'd like to work on this"** (you will not get a response, just go with it) and optionally request that someone works with you (and wait for a second person to accept your request).
2. Use this issue to write down what you need (collect reference material) and discuss, if necessary, what to write and what not to write.
3. Draft a PR and request the feedback from at least one other JavaScript member (you can ping the #maintaining-javascript channel in Slack).
